pandas/_libs/parsers.pyx

```diff
@@ -178,13 +178,13 @@ cdef extern from "parser/tokenizer.h":
         char thousands
 
         int header # Boolean: 1: has header, 0: no header
-        int header_start # header row start
-        int header_end # header row end
+        ssize_t header_start # header row start
+        ssize_t header_end # header row end
```

Comment:
Ah, I see.  Also, I think in light of what @wesm said <a href="https://github.com/pandas-dev/pandas/pull/17040#issuecomment-316812706">below</a>, it might be preferable to use `int64_t` instead given the cross-platform compatibility issue.